core/droidkaigiui/src/commonMain/kotlin/io/github/droidkaigi/confsched/droidkaigiui/extension/ModifierExtensions.kt

```diff
@@ -5,10 +5,19 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 
 @Composable
-expect fun Modifier.enableMouseDragScroll(scrollableState: ScrollableState): Modifier
+expect fun Modifier.enableMouseDragScroll(
+    scrollableState: ScrollableState,
+): Modifier
 
 @Composable
 expect fun Modifier.bindMouseBackForward(
     onBackPressed: () -> Unit,
     onForwardPressed: () -> Unit,
 ): Modifier
```

Comment:
If we follow the comments below, should we also rename this method, which has already been merged, in a separate task?
https://github.com/DroidKaigi/conference-app-2025/pull/554#discussion_r2327052954